vital/vital_types.h

```diff
@@ -58,6 +58,9 @@ typedef int64_t track_id_t;
 /// The type of a frame number
 typedef int64_t frame_id_t;
 
+// Time in micro-seconds
+typedef int64_t time_us_t;
```

Comment:
I don't think I'd do it in this commit anyway, but one of these days it would be nice for us to switch to `using time_us_t = int64_t;` :smile:.